packages/contracts/utils/event.ts

```diff
@@ -11,6 +12,20 @@ export async function findEvent<T>(tx: ContractTransaction, eventName: string) {
   return event as T;
 }
 
+export async function findEventTopicLog(
+  tx: ContractTransaction,
+  iface: Interface,
+  eventName: string
+): Promise<LogDescription> {
+  const receipt = await tx.wait();
+  const topic = iface.getEventTopic(eventName);
+  const log = receipt.logs.find(x => x.topics.indexOf(topic) >= 0);
```

Comment:
the event topic is always at index 0 of `x.topics` if it is not there is something wrong

Reply:
Fixed in 3fcc083562c7c4c39812ebafebfd631a50c1724c